application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +146,35 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
-}
+
+    private void reminderFailureAction(int id, long channelId, long authorId,
+                                       CharSequence content, long guildId, int reminderFailureCounter) {
+        if (reminderFailureCounter <= 3) {
```

Comment:
3 is a magic number. put it into a well named constant `MAX_RETRY_ATTEMPTS` instead

Reply:
i had a feeling that was incorrect way of doing it, makes it hard to look for it and change in future.

Reply:
its usually easier for people to adjust constants when they are in the expected place (at the beginning of the file) and not in the middle of the code flow